.github/workflows/configlet.yml

```diff
@@ -0,0 +1,16 @@
+name: Configlet CI
+
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  configlet:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+
+    - name: Fetch configlet
+      uses: exercism/github-actions/configlet-ci@main
```

Comment:
Any reason why we use a branch here, rather than a tag?

To increase reproducibility of individual runs, GH themselves suggest to prefer tags or commits over branches.

Reply:
This means it will just pull the latest from main whatever that is, without you having to update your local version. ie, you always get upstream version of coniglet in CI.

Reply:
The interface of this action won't ever change. It's meant to install configlet and add it to the PATH, that's it. All changes would be internal to the action, so it doesn't really make a difference. It's also fully under our control, so we don't have to use a hash.

Reply:
Which might be undesirable.

Let's consider there is some change in lining. This change would potentially fail today's commits if rerun with the new version.

I'd prefer a major versioned branch over `main` that would of course be bumped correctly if incompatible Libra are introduced

Reply:
The version of the action wouldn't affect it. It installs the latest configlet, no matter what that is. If you want to fix a configlet version, you'd have to download it yourself or through your own action. It's out of scope for `github-actions/configlet-ci`

Reply:
To clarify: the action is meant to be the equivalent to running the `fetch-configlet` script, which doesn't take a version as input either.

I agree it might be useful to pin a track to a specific version of configlet but that would need to be handled elsewhere.